grc/gui_qt/components/window.py

```diff
@@ -1475,6 +1475,28 @@ def exit_triggered(self):
                 # We cancelled closing a tab. We don't want to close the application
                 return
 
+        # Save the panel settings
```

Comment:
Can these use `!x.isHidden()` to avoid the if/else?

Reply:
Yes, that can be done. I aggree, it looks much better. I'll fix it